Description: relax dependency on pretty_assertions
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,7 +19,7 @@
 log = ">= 0.4.11, < 0.5"
 md5 = "0.7.0"
 path-clean = "0.1.0"
-pretty_assertions = "1.1.0"
+pretty_assertions = ">= 0.6.1, < 1.2"
 rayon = ">= 1.4.1, < 1.6"
 serde = { version = ">= 1.0.130, < 1.1", features = ["derive"] }
 thiserror = ">= 1.0.20, < 1.1"
